### PR TITLE
fix: Simplified the error messages and added a bit more info to one of them

### PR DESCRIPTION
**Pull request by John-Memphis**

### CODE REVIEW

File: src/functions.js [hunk 1]
```diff
@@ -87,15 +87,12 @@ async function createFunction(memphisEvent, eventHandler, asJson = false) {
                         headers: processedHeaders,
                         payload: Buffer.from(processedMessage).toString('base64')
                     });
-                } else if (processedMessage === null && processedHeaders === null) {
```
Comment (shay23b) [inline src/functions.js:90]: Why did you remove this part? I think it's still needed and only need to remove the next 'else if'

**Review by shay23b — CHANGES_REQUESTED** [review]

File: src/functions.js (hunk 1 above)
Comment (John-Memphis) [reply src/functions.js:90]: You are correct. I don't know why I removed that one. I didn't do it to the python SDK and I did similar changes. I added it back.

File: src/functions.js [hunk 2]
```diff
@@ -79,23 +79,22 @@ async function createFunction(memphisEvent, eventHandler, asJson = false) {
...
+                    processedMessage = Buffer.from(JSON.stringify(processedMessage), 'utf-8');
                 }
 
                 if (processedMessage instanceof Uint8Array && processedHeaders instanceof Object) {
                     processedEvents.messages.push({
                         headers: processedHeaders,
                         payload: Buffer.from(processedMessage).toString('base64')
                     });
-                } else if (processedMessage === null && processedHeaders === null) {
+                } else if (processedMessage == null && processedHeaders == null ) {
```
Comment (shay23b) [inline src/functions.js]: it should be === instead of ==

**Review by shay23b — CHANGES_REQUESTED** [review]

File: src/functions.js (hunk 2 above)
Comment (John-Memphis) [reply src/functions.js]: Changed

**Review by shay23b — APPROVED** [review]
